src/widgets/member-activity/ui/member-activity.tsx

```diff
@@ -29,19 +30,35 @@ const localizedContent = {
   },
 };
 
-export const LangContext = createContext<'ru' | 'en'>('en');
+const mentorsActivityData = {
+  en: {
+    header: 'Mentor activities',
+    info: 'The main tasks of a mentor for the duration of the course, but if you are willing to give students more - the list is not limited to anything',
+  },
+  ru: {
+    header: 'Деятельность ментора',
+    info: 'Основные задачи ментора на период курса, но если вы готовы дать студентам больше - список не ограничен ни чем',
+  },
+};
```

Comment:
Agreed, but the last meeting decided not to refactor this widget to nextjs merge
I have now separately taken into work isuue with refactoring of the component in FSD and will immediately make edits on this issue as well, because `localizedContent` should also be removed, etc.